simulator.py

```diff
@@ -36,12 +36,20 @@ def fake_sms(
     with_initial_pause=False,
     **kwargs,
 ):
+    additional_args = {
+        "company": user_profile.account_info.get("company", "your company"),
+        "name": "",
```

Comment:
If we don't have a name the message will have a hanging space: `Congrats !`. I dont think we should care